src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3539,10 +3545,10 @@ QImage ViewProviderSketch::renderConstrIcon(const QString &type,
     // Constants to help create constraint icons
     QString joinStr = QString::fromLatin1(", ");
 
-    QImage icon = Gui::BitmapFactory().pixmap(type.toLatin1()).toImage();
+    QImage icon = Gui::BitmapFactory().pixmapFromSvg(type.toLatin1().data(),QSizeF(constraintIconSize,constraintIconSize)).toImage();
 
     QFont font = QApplication::font();
-    font.setPixelSize(11);
+    font.setPixelSize(constraintIconSize * 0.8);

```

Comment:
I'd rather make the `contraintIconSize` related to the font size:
```
int dpi = QApplication::desktop()->logicalDpiX();
float fontSizePt = QApplication::fontMetrics().height();
float fontSizePx = fontSizePt * dpi * devicePixelRatio;
float constraintIconSize = fontSize * 1.25;
font.setPointSize(fontSizePt);
```

Also, it seems that constraintIconSize * 0.8 means a default application font. Maybe `setPixelSize` isn't needed at all?

Reply:
It is other font - used for constraint index (subscript). It should be related Iconsize.

Reply:
Well, in that case let's just make the constraint index font relative to the sketcher font size:

```
sketcherFontSizePx = getSketcherFontSizePx();
constraintSubscriptFontSizePx = sketcherFontSizePx * 0.8;

constraintIconSize = sketcherFontSizePx;

```